src/app/pages/ParaTimesPage/TransactionAmount/FeesSection.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import { Box, Button, Collapsible, FormField, TextInput } from 'grommet'
+import { useTranslation } from 'react-i18next'
+
+type FeesSectionProps = {
+  feeAmount: string
+  feeGas: string
+  ticker: string
+}
+
+export const FeesSection = ({ feeAmount, feeGas, ticker }: FeesSectionProps) => {
+  const { t } = useTranslation()
+  const [open, setOpen] = useState(false)
+
+  return (
+    <Box pad={{ vertical: 'medium' }}>
+      <Box align="start">
+        <Button
+          label={t('paraTimes.amount.advanced', 'Advanced')}
+          margin={{ bottom: 'medium' }}
+          onClick={() => setOpen(!open)}
+          secondary
+          size="small"
+        />
+      </Box>
+      <Collapsible open={open}>
+        <FormField
+          name="feeAmount"
+          validate={{
+            regexp: /^(\s*|[0-9][0-9]*)$/,
+            message: t('paraTimes.validation.invalidFee', 'Value must be integer greater than or equal to 0'),
```

Comment:
Pretty sure feeAmount is in ROSE units, and input should allow decimal places.
Also add `step="any" min="0"` to number input

feeGas seems to use (paratime's?) base units. I would vote for consistency with feeAmount.

@pro-wh did extension have fee inputs in base units for good reason?
And setFeeAmount uses paratime's denomination, but what does setFeeGas use?

Reply:
dunno, but it seems nice not to have to type 0.00000000 all the time

Reply:
oh i was wrong, feeAmount really is in nano ROSE, and feeGas is in "gas units" (according to docs https://docs.oasis.dev/oasis-core/consensus/transactions/#gas)

Can add `step="1" min="0"` to number input